src/conf/opts.c

```diff
@@ -384,12 +384,12 @@ static void check_dpms_conf(dpms_conf_t *dpms_conf) {
         dpms_conf->disabled = true;
     }
     
-    if (dpms_conf->timeout[ON_AC] <= conf.dim_conf.timeout[ON_AC]) {
+    if (!conf.dim_conf.disabled && dpms_conf->timeout[ON_AC] <= conf.dim_conf.timeout[ON_AC]) {
```

Comment:
I'd rather make an unique check instead of checking if dim is enabled twice, ie: 
```
if (!conf.dim_conf.disabled) { 
    if (dpms_conf->timeout[ON_AC ...) { ... } 
    if (dpms_conf->timeout[ON_BATTERY] { ... } 
}
```

Reply:
Ha, that was exactly my first idea, but then I decided to keep the checks atomic ^^

I'll update the PR in a bit!